### DISCUSSION TITLE
KQC user community

### DISCUSSION THREAD

**Discussion by Qvadi** [iqm-finland/KQCircuits#68] [Q&A] [answered]
Hi there,
I wonder if this is the only forum to discuss KQCircuits and its use-cases with the community? 
Is there a slack channel for eg?

**Comment by qpavsmi** [comment, marked as answer]
Hello,
We were thinking to sticking to Github Q&A pages, as we're unsure about dedicating some of our bandwidth to moderating a discussion channel.

We do have a Discord channel that we have used for our 5-lecture winter school event, seems that there are still people in there writing stuff once in a while. We haven't planned to support it officially just yet, but if you'd like to jump in anyway, email me at pavel.smirnov@meetiqm.com and I'll send you an invite link
